### PR TITLE
[Upcoming] Release 1.2.0

### PR DESCRIPTION
**Pull request by skoudoro**
# Summary

Preparation for 1.2.0 release, targeting Monday, April 13. 

# Desired PR (need review / feedback / update)

- [x] #2108 (Submitter: @skoudoro)
- [x] #2103  (Submitter: @ShrishtiHore )
- [x] #2091 (Submitter: @ShubhamShaswat)
- [x] #2088 (Submitter: @BramshQamar )
- [x] #2086 (Submitter: @aryanshomray )
- [x] #2057 (Submitter: @jhlegarreta)
- [x] #2001 (Submitter: @Garyfallidis )
- [x] #1861 (Submitter: @arokem )
- [x] #2124 (Submitter: @BramshQamar )

A lot of PR's, but most of them can be done on time, I think :smile:

Time for a push, DIPY Team!!

Please comment to flag any other issues or PR that should be addressed for this release

# Release checklist
- [x] add release_notes 1.2.0
- [x] Update changelog and api_changes
- [x] Update .mailmap
- [x] Set release number in info.py and doc/conf.py
- [x] Check the copyright years in doc/conf.py and LICENSE
- [x] Check documentation
- [x] Check Tutorial
- [x] Check unit test and USE-PRE
- [x] Update index.rst
- [x] Update Highligth.rst, old_news.rst, stateofheart.rst
- [x] Update developers.rst list
- [ ] Update website
- [x] Update deprecation cycle

### CODE REVIEW

**Comment by skoudoro** [comment]
Can you have a look @arokem and @Garyfallidis and then, you can merge

**Comment by Garyfallidis** [comment]
Looks good. Don't forget to add Changelog for 1.2.0 too.

File: Changelog [hunk 1]
```diff
@@ -24,6 +24,38 @@ Dipy
...
+- Added b-tensor encoding for gradient table.
+- Stats module refactored.
+- Numpy minimum version is 1.2.0.
+- Fixed compatibilities with FURY 0.6+, VTK9+, CVXPY 1.1+.
+- Added multiple tutorials for DIPY command line interfaces.
+- Updated SH basis convention.
+- Improved performance of tissue classification.
+- Fixed a memory overlap bug (multi_median).
+- Large documentation update (typography / references).
+- Closed 126 issues and merged 50 pull requests.
```
Comment (Garyfallidis) [inline Changelog:39]: Same as in 1.1.0?

Comment (skoudoro) [reply Changelog]: good catch! fixed

File: Changelog [hunk 2]
```diff
@@ -24,6 +24,38 @@ Dipy
 
 The code found in Dipy was created by the people found in the AUTHOR file.
 
+* 1.2.0 (Tuesday, September 9, 2020)
+
+- New command line interfaces for Group Analysis: BUAN.
```
Comment (arokem) [inline Changelog:29]: => “group analysis” does not require capitalization

File: Changelog [hunk 3]
```diff
@@ -24,6 +24,38 @@ Dipy
...
+- Added multiple tutorials for DIPY command line interfaces.
+- Updated SH basis convention.
+- Improved performance of tissue classification.
+- Fixed a memory overlap bug (multi_median).
+- Large documentation update (typography / references).
+- Closed 255 issues and merged 92 pull requests.
+
+* 1.1.1 (Friday, January 10, 2020)
+
+- New module for deep learning ``DIPY.NN`` (uses TensorFlow 2.0).
```
Comment (arokem) [inline Changelog:43]: Maybe lower case? ‘dipy.nn’?

File: Changelog [hunk 4]
```diff
@@ -24,6 +24,38 @@ Dipy
...
+- Improved DKI performance and increased utilities.
+- Non-linear and RESTORE fits from DTI compatible now with DKI.
+- Numerical solutions for estimating axial, radial and mean kurtosis.
+- Added Kurtosis Fractional Anisotropy by Glenn et al. 2015.
+- Added Mean Kurtosis Tensor by Hansen et al. 2013.
+- Nibabel minimum version is 3.0.0.
+- Azure CI added and Appveyor CI removed.
+- New command line interfaces for LPCA, MPPCA and Gibbs Unringing.
```
Comment (arokem) [inline Changelog:51]: “unringing” also lower case

File: Changelog [hunk 5]
```diff
@@ -24,6 +24,38 @@ Dipy
...
+- New MTMS CSD tutorial added.
```
Comment (arokem) [inline Changelog:52]: Typo: “MSMT”

File: doc/index.rst [hunk 6]
```diff
@@ -6,33 +6,29 @@ Diffusion Imaging In Python
...
-- New command line interfaces for LPCA, MPPCA and Gibbs Unringing.
-- New MTMS CSD tutorial added.
-- Horizon refactored and updated to support StatefulTractograms.
-- Speeded up all cython modules by using a smarter configuration setting.
-- All tutorials updated to API changes and 2 new tutorials added.
-- Large documentation update.
-- Closed 126 issues and merged 50 pull requests.
+**DIPY 1.2.0** is now available. New features include:
+
+- New command line interfaces for Group Analysis: BUAN.
```
Comment (arokem) [inline doc/index.rst:21]: Here as well capitalization

File: doc/old_highlights.rst [hunk 7]
```diff
@@ -4,6 +4,25 @@
...
+- Added Mean Kurtosis Tensor by Hansen et al. 2013.
+- Nibabel minimum version is 3.0.0.
+- Azure CI added and Appveyor CI removed.
+- New command line interfaces for LPCA, MPPCA and Gibbs Unringing.
+- New MTMS CSD tutorial added.
+- Horizon refactored and updated to support StatefulTractograms.
+- Speeded up all cython modules by using a smarter configuration setting.
+- All tutorials updated to API changes and 2 new tutorials added.
+- Large documentation update.
+- Closed 126 issues and merged 50 pull requests.
```
Comment (arokem) [inline doc/old_highlights.rst:23]: Same comments as above apply here

**Review by arokem — COMMENTED** [review]
Awesome! Just a few small things

**Comment by skoudoro** [comment]
Thank you for the feedback @arokem and @Garyfallidis! 

All comments are addressed and this is ready to go. Tests are passing and doc is ok. we just need to merge this and tag the release

**Comment by arokem** [comment]
Looks good. Let's go!

**Pull request merged**